fixtures/clients.yaml

```diff
@@ -0,0 +1,23 @@
+App\Entity\Client:
+    client_{1..10}:
+      name: 'client_<current()>'
+      description: <text()>
+      isActive: <boolean()>
+      quota: -1
+      owner: '@user'
+      maxParallelJobs: 1
+      
+    client_to_delete:
+      name: 'client_to_delete'
+      description: <text()>
+      isActive: <boolean()>
+      quota: -1
+      owner: '@user'
+      maxParallelJobs: 1
```

Comment:
client_to_delete ? Will it be deleted during a test?

Haven't we agreed to try to make tests as autonomous and self sustained as possible ?
Haven't we agreed not to fixturize any content that would be changed in the tests?

I know that right now we are resetting all our DB and fixtures for each execution of the DB but once our test cases increase we might want to separate the resetting of the database and the execution of them because the developer might just want to execute a single test or a single suite repeatedly while developing them. Well in fact I'm sure you have already experienced the inconvenience of having to wait for the DB to recreate just to check one test.

Please review all the cases where you are changing preloaded content (deletes and updates).

Reply:
Ok. And in the cases that the request should fail and it should not change the content, should I use the preloaded content, or it is not convenient either?

Reply:
If it is not a big deal I think is safer to not use it either in that case. If there is a mistake and it does change it we might end up with misleading results from other tests that also used the fixtured object.